mal/utils.py

```diff
@@ -76,6 +77,25 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def checked_cancer(func):
```

Comment:
As you wish @lubien:

![image](https://user-images.githubusercontent.com/7642878/45521436-6c5c1780-b7ad-11e8-9b61-64552a41e4bf.png)